GVFS/GVFS.FunctionalTests/Categories.cs

```diff
@@ -18,8 +18,23 @@ public static class MacTODO
             // Tests that require #360 (detecting/handling new empty folders)
             public const string NeedsNewFolderCreateNotification = "NeedsNewFolderCreateNotification";
 
-            // Tests for GVFS features that are not required for correct git functionality
-            public const string M4 = "M4_GVFSFeatures";
+            // Tests that require the Status Cache to be built
+            public const string NeedsStatusCache = "NeedsStatusCache";
+
+            // Tests that require Config to be built
+            public const string NeedsGVFSConfig = "NeedsConfig";
+
+            // Tests that require VFS Service

```

Comment:
super nit: The comments on some of these exclusion strings aren't proper English. The `VFS Service` could refer to the PrjFS logging daemon (since it's a service in our Virtual File System component) or it could refer to the Service that lives in the VFS4G C# codebase (which is what you mean).

Ultimately, we expect these to go away soon (yay!) so I don't know if it's worth fixing, but I prefer proper English for comments :)

Reply:
I want to get this PR in early today to see if anything turns flakey etc.. while i'm in the office.  I'm definitely up for better comments in a followup PR.  I want it to be clear!